src/libraries/System.Text.Encodings.Web/src/System/Text/Encodings/Web/AdvSimdHelper.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+#if NETCOREAPP
+using System.Runtime.Intrinsics.Arm;
+#endif
+#if NETCOREAPP5_0
```

Comment:
>NETCOREAPP5_0 [](start = 4, length = 13)

where these defines are defined? I am not seeing it in the csproj at all. is this something you are going to add later?

Reply:
I believe those are baked in by the sdk. In any case, I will probably change this code and shim the intrinsics APIs instead.